css-two-column-layout/styles.css

```diff
@@ -0,0 +1,75 @@
+* {
+  box-sizing: border-box;
+}
+
+.container {
+  max-width: 800px;
+  margin-left: auto;
+  margin-right: auto;
+  font-family: 'Karla', sans-serif;
+}
+
+.column-full {
+  text-align: center;
+  width: 100%;
+}
+
+.header {
+  text-transform: uppercase;
+  letter-spacing: 0.2rem;
+}
+
+.header-text {
+  padding: 0 110px;
```

Comment:
Is this to center the text? You should use `text-align: center`, avoid large padding and margin values.

Reply:
It was to make the text under the header to not take up the whole width and be on two lines.